### ISSUE TITLE
[Bug] Laplace Mechanism File Writing Error

### ISSUE THREAD

**Issue by Quantalabs** [Quantalabs/DiffPriv#26]
**Describe the bug**
Writes to a file with the privatized data like:
```csv
[a, b, c, d]
```
instead of:
```csv
a, b, c, d
```
**To Reproduce**
Try to use some example data:
```python
from DiffPriv import diff as private

def avg(arr):

    sum = 0
    for i in arr: sum += i
    return sum / len(arr)

file = open('data.csv', 'r')
private.lapmech(file, 'output.csv', 1.0, avg)
```
With `data.csv`:
```csv
2,2,2,2,2,1,3,4
```
and check the output at `output.csv`, and you get it as if it were an array and not in `.csv` format.

**Expected behavior**
It should output something like:
```csv
a, b, c, d
```
and not:
```csv
[a, b, c, d]
```

**Anaconda**
- **Anaconda Version** - conda 4.10.3
- **Operating System** - Windows
- **DiffPriv Version** - 1.0.2, latest

**Additional context**
This is for the `laplech` mechanism, but hasn't been checked for the exponential mechanism, so it is possible the issue is there too.

Work will on branch `patch-#26` and will be merged into `main`, but the commit (be it one commit or a squash commit) will be checked out to `maintenance/v1.0.x`, and the target for the patch release will also be on `maintenance/v1.0.x`.

**Comment by Quantalabs** [comment]
As stated, I'll start work on this in `patch-#26`, but external contributions via forks are welcome.

/claim

**Comment by Quantalabs** [comment]
I can confirm that it also does do this with the exponential mechanism. If you run the tests:
```sh
pytest tests/private_test.py
```
The output file from the exponential mechanism, `em-encrypted_test.csv` shows the list brackets.

**Comment by quantum9Innovation** [comment]
Simple fix to this is to combine the array this way:
```python
 " ".join(str(x) for x in output_array)
```
(*From Stack Overflow answer <https://stackoverflow.com/questions/5618878/how-to-convert-list-to-string/5618910#5618910>*)

**Comment by Quantalabs** [comment]
Almost there:
- [x] Laplace mechanism
- [ ] Exponential mechanism

**Comment by Quantalabs** [comment]
Whoops, accidental

**Comment by Quantalabs** [comment]
The PR #27 has been submitted, this fixes the issue and will close it once merged, at which point the squash commit will be checked out onto `maintenance/v1.0.x` and version info will be changed, and then the `v1.0.3` release will be published with the `maintenance/v1.0.x` branch as the target. @quantum9Innovation we're waiting on your review.
